### PR TITLE
Add lobby tags docs, add links to tiers and regions

### PR DESCRIPTION
**Pull request by MasterPtato**
Fixes WEB-46
Fixes WEB-71

### CODE REVIEW

**Comment by MasterPtato** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #87** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/87?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/site/87?utm_source=stack-comment).

File: src/pages/docs/dynamic-servers/concepts/available-tiers.mdx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Available tiers
 
-- `basic-1d16`: 1/16 core, 125 MB RAM
 - `basic-1d8`: 1/8 core, 250 MB RAM
```
Comment (NathanFlurry) [inline src/pages/docs/dynamic-servers/concepts/available-tiers.mdx:3]: Are these values accurate still? I know we updated the pricing page correctly.

File: src/pages/docs/matchmaker/concepts/lobby-tags.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Lobby tags
+
+Lobby tags are an optional property that can be passed to the [`lobbies.find`](/docs/matchmaker/api/lobbies/find), [`lobbies.join`](/docs/matchmaker/api/lobbies/join), and [`lobbies.create`](/docs/matchmaker/api/lobbies/create) endpoints to allow filtering which lobbies the matchmaker will route a player to.
```
Comment (NathanFlurry) [inline src/pages/docs/matchmaker/concepts/lobby-tags.mdx:3]: Does lobbies.join use tags? How would that work?

File: src/pages/docs/matchmaker/concepts/lobby-tags.mdx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+To enable tags, set [`taggable`](http://localhost:3000/docs/general/config#matchmaker-game-modes-game-mode-taggable) to true.
+
+When passed to [`lobbies.create`](/docs/matchmaker/api/lobbies/create), the newly created lobby will have the given tags.
```
Comment (NathanFlurry) [inline src/pages/docs/matchmaker/concepts/lobby-tags.mdx:7]: I think this needs to clarify lobbies.find will set the tags too, right? I know you go in to this below in the example, but thrown off by this.

File: src/pages/docs/matchmaker/concepts/lobby-tags.mdx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+5. **A [`lobbies.find`](/docs/matchmaker/api/lobbies/find) request with the tags `map = tropic`**
+
+   The matchmaker would error because there are no lobbies with the given tags.
+
+<Note>
+  This is assuming
+  [`prevent_auto_create_lobby`](http://localhost:3000/docs/matchmaker/api/lobbies[`lobbies.find`](/docs/matchmaker/api/lobbies/find)#prevent-auto-create-lobby)
+  is `true`. If auto creating lobbies is enabled, cases 4 and 5 would create a new lobby with the given tags
+  and cases 1-3 would create a new lobby with the given tags only if all available servers are full.
```
Comment (NathanFlurry) [inline src/pages/docs/matchmaker/concepts/lobby-tags.mdx:56]: Can we move this at the top? I think talking about "cases 4 and 5" adds more confusion and things to think about. Just something like "uses prevent auto create lobby to demonstrate how lobbies are selected"

**Review by NathanFlurry — CHANGES_REQUESTED** [review]

**Review by NathanFlurry — APPROVED** [review]

**Pull request merged**